src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespacearound/InputWhitespaceAroundAllowEmptyTypesAndNonEmptyClasses2.java

```diff
@@ -23,29 +23,30 @@
 import java.util.function.Function;
 import java.util.function.Supplier;
 
-public class InputWhitespaceAroundAllowEmptyTypesAndNonEmptyClasses2{ // violation
+public class
+    InputWhitespaceAroundAllowEmptyTypesAndNonEmptyClasses2{ // violation ''{' is not .* whitespace'
 
     private Object object;
 
-    class SomeClass{ // violation
+    class SomeClass{ // violation ''{' is not preceded with whitespace'
         int a = 5;
     }
 
-    public class CheckstyleTest{ // violation
+    public class CheckstyleTest{ // violation ''{' is not preceded with whitespace'
         private static final int SOMETHING = 1;
     }
 
-    class MyClass{ int a; } // violation
+    class MyClass{ int a; } // violation ''{' is not preceded with whitespace'
 
     class SomeTestClass{int a;} // 3 violations
 
-    class TestClass { int a; }int b; // violation
+    class TestClass { int a; }int b; // violation ''}' is not followed by whitespace'
 
     class Table {}
 
     interface SupplierFunction<T> extends Function<Supplier<T>, T> {}
 
-    class NotEmptyClass{ public void foo1() { foo2(); } } // violation
+    class NotEmptyClass{ public void foo1() { foo2(); } } // violation ''{' is not .* whitespace'
```

Comment:
Shorten name of class, remove not required visibility modifiers, to give message whole space

Reply:
done